Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-1151-more.dfy.expect

```diff
@@ -6,19 +6,15 @@ git-issue-1151-more.dfy(13,5): Error: Abstract type ('_module.Opaque') cannot be
 
 Dafny program verifier did not attempt verification
 git-issue-1151-more.dfy(13,5): Error: Abstract type ('_module.Opaque') cannot be compiled; perhaps make it a type synonym or use :extern.
-Wrote textual form of partial target program to git-issue-1151-more-java/git_issue_1151_more.java
```

Comment:
Was it intentional to lose these outputs?

Reply:
Completely agree with that and was tempted to do this myself in the past, just didn't spot the connection - thanks for that change then!